src/orchestration/Orchestration.ts

```diff
@@ -501,4 +505,13 @@ export class Orchestration {
             }
         };
     }
+
+    private validateConfiguration(config: Config) {
+        const awsReleaseIdTest = new RegExp(/^[a-zA-Z0-9_\-:/\.]+$/);
+        if (!awsReleaseIdTest.test(config['aws:releaseId'])) {
+            throw new Error(
+                `AWS RUM Client invalid aws:releaseID: ${this.config['aws:releaseId']} must match ${awsReleaseIdTest}`
+            );
+        }
+    }
 }
```

Comment:
blocker: We should also update `meta-data.json` to add explicitly add this new field. We can add data validation into the schema if we want.

Reply:
Updated metadata.json